src/SpectatorView.Unity/Assets/SpectatorView/Scripts/StateSynchronization/NetworkManager.cs

```diff
@@ -142,27 +146,33 @@ protected override void OnDestroy()
                 connectionManager.OnReceive -= OnReceive;
             }
 
+            if (connectionManagerGameObject != null)
+            {
+                connectionManager = null;

```

Comment:
Why are we doing it conditionally here (as opposed, for example, the block above)?

Reply:
good call out.